app/build.gradle.kts

```diff
@@ -49,6 +49,11 @@ dependencies {
     implementation(libs.androidx.ui.graphics)
     implementation(libs.androidx.ui.tooling.preview)
     implementation(libs.androidx.material3)
+    implementation(libs.gson)
+    implementation(libs.jackson.databind)
```

Comment:
Would it be possible to use gson library for conversion?

Reply:
@bluetoothfx Implementation updated.